python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4781,6 +4781,90 @@ def test_rowwise_ops_nullable_int_dtypes(op, expected):
     assert_eq(got, expected)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[ms]"
+            ),
+        },
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[ns]"
+            ),
+            "t3": gd.Series(
+                ["1960-08-31 06:00:00", "2030-08-02 10:00:00"], dtype="<M8[s]"
+            ),
+        },
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[us]"
+            ),
+        },
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[ms]"
+            ),
+            "i1": gd.Series([1001, 2002], dtype="int64"),
+        },
+        {
+            "t1": ["2020-08-01 09:00:00", "1920-05-01 10:30:00"],
+            "t2": ["1940-08-31 06:00:00", pd.NaT],
+        },
+        {
+            "t1": ["2020-08-01 09:00:00", "1920-05-01 10:30:00"],
+            "t2": ["1940-08-31 06:00:00", pd.NaT],
+            "i1": [1001, 2002],
+        },
+    ],
+)
+@pytest.mark.parametrize("op", ["max", "min"])
+@pytest.mark.parametrize("skipna", [True, False])
+@pytest.mark.skipif()
+def test_rowwise_ops_datetime_dtypes(data, op, skipna):
+
+    if (
+        data
+        == {
+            "t1": ["2020-08-01 09:00:00", "1920-05-01 10:30:00"],
+            "t2": ["1940-08-31 06:00:00", pd.NaT],
+        }
+        and op == "max"
+        and skipna
+    ):
+        pytest.skip(
```

Comment:
The problem with skipping or xfailing here is it ends up just getting swept under the carpet. Ideally we could split those tests off and assert that Pandas is returning an erroneous result so that when it's fixed we can fix the test as expected.